README.md

```diff
@@ -321,6 +321,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Sheetson](https://sheetson.com) - Instantly turn any Google Sheets into RESTful API. Free plan available.
   * [Shipyard](https://www.shipyardapp.com) — Low-code data orchestration platform for the cloud. Build with a mix of low-code templates and your code (Python, Node.js, Bash, SQL). Our free developer plan offers 10 hours of runtime every month for one user - more than enough to automate multiple workflows.
   * [shrtcode API](https://shrtco.de/docs) - Free URL Shortening API without authorization and no request limits.
+  * [SearchApi](https://www.searchapi.io/) - SearchApi is a robust real-time SERP API delivering structured data from Google and other search engine results pages. The free plan allows for 100 API calls each month.
```

Comment:
100 free calls a month is just not useful, you couldnt develop a application with just 100 calls, basic trial and error of using a new API and exploring its features would exahaust that.

So I wont be adding this one, thank youi!

Reply:
Thanks for the review.

Saw that there are multiple other APIs with 100 free requests/actions per month already listed (even similar)? What makes the threshold?

Reply:
It just doesn’t seem useful to me?

You are welcome to remove others we missed this on of course.

Reply:
Hmm. The API structures Google Search, Google Shopping, Google Jobs, and other Google product results into JSON. It avoids maintaining parsers and dealing with proxies/captchas since it has an automatic retry mechanism on failure. I see many use cases from SEO (such as keyword tracking) to LLM models (real-time searching, news, shopping etc..)

Reply:
The service is useful I do not doubt that. 

100 API calls is not.